src/client/automation/playback/move/event-sequence/event-behaviors.js

```diff
@@ -27,12 +27,10 @@ export class MoveBehaviour {
         while (currentParent && currentParent !== commonAncestor) {
             mouseenterElements.push(currentParent);
 
-            currentParent = nativeMethods.nodeParentNodeGetter.call(currentParent);
+            currentParent = domUtils.getParentExceptShadowRoot(currentParent);
```

Comment:
Please add small description to the PR to clarify why we need to ignore parent shadow root here

Reply:
Shadow root is a document fragment.
```
// div1(HTMLDivElement) -> shadow-root(DocumentFragment) -> div2(HTMLDivElement)
domUtils.getParentExceptShadowRoot(div2); // return div1
```
Do you mean to add description to the first PR message?

Reply:
Yes

Reply:
done